_database_api/status.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Instance Status
+layout: default
+---
+
+# Instance Status
+
+--- row ---
+
+## Get Database Instances Status
+
+--- row ---
+
+`GET https://$DB_API_URL/api/databases/[:database_id]/instances_status`
+
+{:.table}
+| field | type   | description                              |
+| --    | --     | --                                       |
+| id    | string | Instance ID                              |
+| type  | string | Instance type                            |
+| status| string | Instance status                          |
+| role  | string | Instance role in the cluster             |
+
+The request provides information about the instances of a given database.
+Type is about the instance type, so either "db-node" or "gateway" node. An additional service type "utility" can exist.
+Status is about the instance current status. The functional status is "running". Others possibilities are "booting", "restarting", "migrating", "upgrading", "stopped", "removing".
+Role is about the instance role in the cluster. For DB nodes, it will depends of the DB type. Generic values are "leader" and "follower".
```

Comment:
```suggestion
Role is about the instance role in the cluster. For DB nodes, it will depend of the DB type. Generic values are "leader" and "follower".
```